lib/tasks/sandbox_data.rake

```diff
@@ -1,83 +1,17 @@
 # frozen_string_literal: true
 
-namespace :lead_provider do
-  desc "create seed mentors for API testing"
-  task seed_mentors: :environment do
+require "tasks/valid_test_data_generator"
+
+namespace :lead_providers do
+  desc "create seed schools and participants for API testing"
+  task seed_schools_and_participants: :environment do
     logger = Logger.new($stdout)
     logger.formatter = proc do |_severity, _datetime, _progname, msg|
       "#{msg}\n"
     end
 
-    providers_names = ["Capita", "Teach First", "UCL Institute of Education", "Best Practice Network", "Ambition Institute", "Education Development Trust"]
-    cohort_2021 = Cohort.find_by!(start_year: "2021")
-
-    providers_names.each_with_index do |provider_name, index|
-      logger.info("Generating seed mentors for '#{provider_name}'...")
-      lead_provider = LeadProvider.find_by!(name: provider_name)
-      school = create_school_and_associations(lead_provider, cohort_2021, index)
-      generate_mentors(lead_provider, school, cohort_2021, logger)
+    ["Capita", "Teach First", "UCL Institute of Education", "Best Practice Network", "Ambition Institute", "Education Development Trust"].each do |provider|
+      ValidTestDataGenerator::LeadProviderPopulator.call(name: provider, total_schools: 100, participants_per_school: 100)
```

Comment:
So it's like 60k participants in total, right?

Reply:
Yes. 10k for each provider.